src/BuildScriptGenerator/RunScriptGeneratorContext.cs

```diff
@@ -0,0 +1,30 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+namespace Microsoft.Oryx.BuildScriptGenerator
+{
+    /// <summary>
+    /// Options for the run script generation process.
+    /// </summary>
+    public class RunScriptGeneratorContext : ScriptGeneratorContext
+    {
+        /// <summary>
+        /// Gets or sets the name of the main programming language used in the repo.
+        /// If none is given, a language detection algorithm will attemp to detect it.
+        /// </summary>
+        public string Language { get; set; }
```

Comment:
we like to use "platorm" instead of "language", although we might still have occurrences of "language" in the code. The reason for this is node for instance is not really a language but a "programming platform".